### PR TITLE
scratchblocks: use v3.0 with only nb as language

### PR DESCRIPTION
**Pull request by arve0**
renders to svg

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -76,7 +75,7 @@ gulp.task('css', function (cb) {
 gulp.task('assets', function () {
   return gulp.src([
     'assets/**/*',
-    'node_modules/scratchblocks2/build/*/*.png',
+    'node_modules/scratchblocks/src/block_images/*.png',
```
Comment (tjvr) [inline gulpfile.babel.js]: FWIW, `block_images` was removed in v3 (as was the jQuery dependency). :)

Comment (arve0) [inline gulpfile.babel.js]: @tjvr Thanks for letting me know!

**Comment by arve0** [comment]
[Live build](http://arve0.github.io/oppgaver/scratch/).

Waiting for feedback in tjvr/scratchblocks#133.

@gahjelle Can you take a look, and check if the blocks renders correctly?

**Comment by gahjelle** [comment]
Det ser ganske lovende ut.

Men i tillegg til problemene med operator-klossene som du refererer til (ser ut som om det gjelder alle operator-klosser) er det noen klosser som rendres på engelsk. For eksempel `når grønt flagg klikkes` og `vend høyre () grader` i http://arve0.github.io/oppgaver/scratch/astrokatt/astrokatt.html.

I tillegg ser det ut til at den alternative skrivemåten `vend @ () grader` som brukes blant annet i http://arve0.github.io/oppgaver/scratch/labyrint/labyrint.html ikke lenger er støttet? Det vil uansett ikke være et stort problem, vi kan bare oppdatere kildekoden til den mer lesbare varianten :)

**Comment by arve0** [comment]
**Remaining**
- [ ] operators rendering properly
- [ ] when flag clicked translation
- [ ] update code, `vend @` to `vend venstre`, see [translation.js line 1198](https://github.com/arve0/scratchblocks/blob/master/src/translations.js#L1198)

**Comment by gahjelle** [comment]
Ah ... the translations.js file has changed, this is at least much more readable :)

Hmm ... seems to be related to the alias-thing, but it seems that the exact phrases that are used in the blocks source code are not in the `commands`-part of the translations.js-file. Could test what happens if they are added there? Or if we update the blocks source code to use `når @greenFlag klikkes`? Although I am not totally sure if the `@greenFlag` construct has a special meaning?

**Comment by arve0** [comment]
[New build](http://arve0.github.io/oppgaver/), with kodeklubben/oppgaver#228.

**Comment by arve0** [comment]
Quite some stuff [happening at scratchblocks](https://github.com/tjvr/scratchblocks/commits/master). 3.1 is not released yet, but seems like most of our issues have been resolved.

See the [updated build](https://arve0.github.io/oppgaver/scratch).

@gahjelle If you do not find any more errors, I think this is good to merge.

**Comment by tjvr** [comment]
@arve0 Thanks for the prompt—I've just gone ahead and released v3.1. Let me know how you get on!

**Comment by gahjelle** [comment]
As far as I can tell the updated build is beautiful! :+1:

**Pull request merged**